app/controllers/wishlists_controller.rb

```diff
@@ -69,4 +61,11 @@ def update
     def wishlist_params
         params.require(:wishlist).permit(:title, :description, :color, :private, :organization_id)
     end
+
+    def set_organization
+      subdomain = request.subdomain
+      @organization = Organization.find_by(subdomain:)
+
+      redirect_to root_url(subdomain: 'www'), allow_other_host: true unless @organization
```

Comment:
Sorry, this works but there's another error 😅 
the redirect is going to the `root_url` which is going to the `passthrough#index`, and as user it redirects to `wishlists_path` (which requires a domain, therefore it breaks)

<img width="1511" alt="Screenshot 2023-09-19 at 23 00 42" src="https://github.com/wyshlist/wyshlist/assets/74747669/2424f370-c63a-4d8a-b673-70983c8185bc">

Reply:
already talked about it in slack :) users always will go to a subdomain. We will tackle it when we want to add wishlists to our own platform. Out of scope for this sprint.